ods_ci/tests/Resources/CLI/ModelServing/llm.resource

```diff
@@ -592,3 +601,14 @@ Clean Up Test Project
     ELSE
         Log    Project deletion started, but won't wait for it to finish..
     END
+
+Create PVC And Download Model From S3
+    [Arguments]    ${model_name}    ${bucket_name}    ${endpoint}
+    ...            ${region}    ${access_key_id}    ${access_key}
+    ...            ${use_https}    ${namespace}    ${downoad_timeout}=300s
+    Create File From Template    ${DOWNLOAD_PVC_FILEPATH}    ${DOWNLOAD_PVC_FILLED_FILEPATH}
+    ${rc}    ${out}=    Run And Return Rc And Output    oc -n ${namespace} apply -f ${DOWNLOAD_PVC_FILLED_FILEPATH}
+    Should Be Equal As Integers    ${rc}    ${0}
```

Comment:
should we add the check for the PVC ?

Reply:
like checking the disk size occupied?

Reply:
yes check if it is bound or not with PV

Reply:
hm this might be an unstable check I think, cos after this pod finishes the execution the PV gets unbouded